pages/docs/reference/extensions/x.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: 'x' 
-weight: 11
+weight: 12
```

Comment:
why we need a change in weight? why `index.md` can't be 10?

Reply:
I actually don't know how weight attribute works.
to avoid any type of collision I did that.
Sorry for that.